hub/powertoys/command-palette/publish-extension.md

```diff
@@ -245,37 +146,78 @@ After submission, Microsoft reviews your extension for certification. Monitor yo
+
+- **Automatically discoverable**: Users can find and install your extension directly from within Command Palette
+- **Easy to install**: Simple `winget install` command-line installation
+- **Easy to update**: Users can update extensions with `winget upgrade`
+- **Open source**: The WinGet package repository is transparent and community-driven
 
 > [!TIP]
 > **What is WinGet?**
-> WinGet is Microsoft's open-source command-line package manager for Windows. It's similar to package managers like npm or pip, but for Windows applications. When you publish to WinGet, users can install your extension with a simple `winget install` command. It also enables automatic discovery within Command Palette.
+> WinGet is Microsoft's open-source command-line package manager for Windows. It's similar to package managers like npm (Node.js) or pip (Python), but for Windows applications. Publishing to WinGet makes your extension accessible to both command-line users and Command Palette users.
+
+**WinGet manifest requirements**
 
-Before submitting your manifest to WinGet, check the following two requirements:
+Before submitting your extension to WinGet, ensure your manifest meets these two requirements:
 
-**Add the `windows-commandpalette-extension` tag**
+- Add the `windows-commandpalette-extension` tag
 
-Command Palette uses the special `windows-commandpalette-extension` tag to discover extensions. Make sure that your manifest includes this tag so that Command Palette can discover your extension. Add the following code to each `.locale.*.yaml` file in your manifest:
+Command Palette uses the special `windows-commandpalette-extension` tag to discover extensions. Add this tag to each `.locale.*.yaml` file in your WinGet manifest:
 
 ```yaml
 Tags:
 - windows-commandpalette-extension
 ```
 
-**Ensure WindowsAppSdk is listed as a dependency**
+- Ensure WindowsAppSdk is listed as a dependency
 
-If you're using Windows App SDK, make sure that it's listed as a dependency of your package. Add the following code to your `.installer.yaml` manifest:
+If you're using Windows App SDK, list it as a dependency in your `.installer.yaml` manifest:
 
 ```yaml
 Dependencies:
   PackageDependencies:
   - PackageIdentifier: Microsoft.WindowsAppRuntime.#.#
 ```
 
+Replace `#.#` with the specific version of Windows App SDK your extension requires (for example, `1.6`).
+
 ## Guide to WinGet publishing
 
 Publishing to WinGet is the recommended distribution method for Command Palette extensions. It enables automatic discovery and installation directly within Command Palette. This guide covers most of the WinGet publication process, from preparing your project and creating build scripts to setting up GitHub Actions automation and submitting your first package manifest. You'll learn how to create installer packages, configure automated builds, and navigate the WinGet submission workflow to make your extension easily discoverable and installable for users.
 
+```mermaid
```

Comment:
Are mermaid diagrams supported on Learn now, or are you just showing the source in a code block here?